app/controllers/admin/split_details_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::SplitDetailsController < AuthenticatedAdminController
+  def edit
+    @split = Split.find params[:split_id]
+  end
+
+  def update
+    @split = Split.find params[:split_id]
+    if @split.update!(split_context_params)
```

Comment:
Missed this before but we should be using update not update! here otherwise we will never hit the else path. 

Is there no feature spec for this feature? Dun dun dun. I think I missed that before as  well :(